entries/slotfills/deduplication/deduplication.tsx

```diff
@@ -2,23 +2,33 @@ import { ToggleControl } from '@wordpress/components';
 import { PluginDocumentSettingPanel } from '@wordpress/edit-post';
 import { __ } from '@wordpress/i18n';
 import { usePostMetaValue } from '@alleyinteractive/block-editor-tools';
-
-/**
- * The following code is a temporary fix.
- *
- * Once the issue linked below is resolved, this code can be removed.
- * @link https://github.com/alleyinteractive/alley-scripts/issues/473
- */
 import { useSelect } from '@wordpress/data';
+import { useEffect } from '@wordpress/element';
```

Comment:
We should import from React instead. The WP project has transitioned away from using `@wordpress/element` to using React directly.